README.md

```diff
@@ -19,10 +19,10 @@ end
 By default **you don't need Rust installed** because the lib will try to download
 a precompiled NIF file. In case you want to force compilation set the
 `HTML5EVER_BUILD` environment variable to `true` or `1`. Alternatively you can also set the
-application env `:skip_compilation?` to `false` in order to force the build:
+application env `:build_from_source` to `true` in order to force the build:
 
 ```elixir
-config :html5ever, Html5ever.Native, skip_compilation?: false
+config :html5ever, Html5ever, build_from_source: true
```

Comment:
Don't you want to keep it as `Html5ever.Native` because that's likely the module we would use if we move this to Rustler? :)

Reply:
Or keep it as is and we can change later (when we move it to Rustler).

Reply:
Yeah, I will leave as it is because I need to change Rustler in order to use the same module from the NIF. I will change that in the future.